lib/WasmReader/WasmBytecodeGenerator.cpp

```diff
@@ -785,8 +840,8 @@ WasmBytecodeGenerator::GetConstReg(T constVal)
     Js::RegSlot location = m_funcInfo->GetConst(constVal);
     if (location == Js::Constants::NoRegister)
     {
-        WasmRegisterSpace * regSpace = GetRegisterSpace(m_reader->m_currentNode.type);
-        location = regSpace->AcquireConstRegister();
+        WasmRegisterSpace * regSpace = GetRegisterSpace(WasmTypes::I32);
+        location = regSpace->AcquireTmpRegister();
```

Comment:
Ah, great catch. Yes, that was a leftover from my aborted attempt to get GetConst() up and running but it looks like I didn't revert it completely. I've restored it back to its original, albeit incomplete, condition. Thanks!